lookout/style/typos/generation.py

```diff
@@ -419,9 +428,15 @@ def _load_tree(self, tree: dict) -> None:
             for i, s in enumerate(vocab)}
         wv.bucket = self.wv["bucket"]
         wv.index2word = wv.index2entity = vocab
-        wv.num_ngram_vectors = self.wv["num_ngram_vectors"]
         wv.vectors_ngrams = numpy.array(self.wv["vectors_ngrams"])
-        wv.hash2index = {k: v for v, k in enumerate(self.wv["hash2index"])}
+        if "buckets_word" in self.wv.keys() and isinstance(self.wv["buckets_word"], dict):
```

Comment:
`"buckets_word" in self.wv.keys()`. `buckets_word` should be always in a tree.

Reply:
No, it is not for the previous versions of the model (the one that's in prod right now for example), so this `if` is needed for backward compatibility.

Reply:
I see that you set `buckets_word` to None if there is no `buckets_word` does it mean that it is not necessary and model can work without `buckets_word`?

Reply:
Yes, actually it was empty in the last tried versions of the fasttext models. And when I wrote this code a couple of months ago and ran with the previous version of gensim, it was not empty. 
It may be working like some kind of a cache to store stuff now, but I'm not sure.

Tbh it looks like they (gensim) are currently slightly lost in fasttext model storage logic, they have complex inheritance architecture, so it's not ideal. I'm trying to keep all the attributes stored just in case.

Reply:
let's find out if we need `buckets_word ` or remove it.

Reply:
I don't know the architecrute of the gensim fasttext so well and I don't think it's reasonable to waste time on learning it. Imo we should always save all the attributes.

Reply:
ok, then let's remove it and upload the new fixed model.

Reply:
I don't get what you mean

Reply:
remove what is related to old model backward compatibility and upload a new one, please.

Reply:
This way I will just remove the `if "buckets_word" in self.wv.keys()` part, and anyway I think we should have backward compatibility. At least while I can't acess the new model.

Reply:
ok, please add a comment then.